pkgs/development/tools/build-managers/jam/default.nix

```diff
@@ -17,13 +11,13 @@ stdenv.mkDerivation {
 
   installPhase = ''
     mkdir -p $out/bin
-    cp ${bindir}/jam $out/bin
+    cp bin.*/jam $out/bin
   '';
 
   meta = {
```

Comment:
Thanks, here I touched jam only to support darwin, I'm going to update jam in a separate review.